relayer/spy_relayer/src/utils/swim.ts

```diff
@@ -0,0 +1,47 @@
+import {
```

Comment:
nothing to worry about for now but will need to figure out where this should logically go since it'll be needed client side in sdk as well so this code might just end up coming from that.